File: GTTS/INL.TrainingService.Models/GetTrainingEventRosterGroups_Item.cs
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventRosterGroups_Item : IGetTrainingEventRosterGroups_Item
    {
        public string GroupName { get; set; }
        public long TrainingEventGroupID { get; set; }
        public List<GetTrainingEventRoster_Item> Rosters { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-roster-groups_item.ts


import { GetTrainingEventRoster_Item } from './get-training-event-roster_item';
import { IGetTrainingEventRosterGroups_Item } from './iget-training-event-roster-groups_item';

export class GetTrainingEventRosterGroups_Item implements IGetTrainingEventRosterGroups_Item {
  
	public GroupName: string = "";
	public TrainingEventGroupID: number = 0;
	public Rosters?: GetTrainingEventRoster_Item[];
  
}






